### PR TITLE
Upgrade the maven-source-plugin from 2.2.1 to 3.2.1 #1875

### PR DESCRIPTION
**Pull request by Superbbbb**
Fixes strongbox/strongbox#1875.

### CODE REVIEW

**Comment by steve-todorov** [comment]
@Superbbbb could you please check your git configuration? It looks like your email is not properly configured.

**Comment by carlspring** [comment]
@Superbbbb ,

The username under which you're committing and pushing to Github seems to be mis-configured. You will have to look into how to configure your git settings properly, amend the author of the commits and then force push back to the branch. (When your user is properly set up, you see your avatar when against your commits in the commit history, otherwise it's a default grey avatar).

**Comment by Superbbbb** [comment]
@carlspring ,

I have set the name and email address and force push the code to the branch. Do I need to create a new pull request?

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1875-SNAPSHOT</version>
```
Comment (carlspring) [inline pom.xml:9]:
```suggestion
    <version>1.0-SNAPSHOT</version>
```
Reverting this to our version, as it passes our tests.

**Comment by carlspring** [comment]
Thank you! Congratulations on your first pull request! :)

**Pull request merged**